.expeditor/scripts/build_mac_hab_binary.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/local/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/shared_release_habitat.sh
+
+# Get secrets! (our auth token and aws creds should be auto-injected but there's a bug:
+# https://github.com/chef/ci-studio-common/issues/200)
+eval "$(vault-util fetch-secret-env)"
```

Comment:
I'm alway very wary of `eval` statements. What does this return?

Reply:
It exposes and exports the secrets from the `vault-util` utility.

Reply:
More specifically, what does the format look like? I'm trying to understand why it _needs_ to be an `eval` as opposed to running a regular command because `eval` is incredibly easy to mess up in very dangerous ways and should only be used as a las resort.

Reply:
It's going to be exported as such: https://github.com/chef/ci-studio-common/blob/master/cmd/vault-util/commands/fetch-secret-env.go#L66, key value pairs. There is an issue (linked) where it doesn't behave the same way on linux as it does on macos.

Reply:
I see. That's fairly reasonable as `eval` calls go. Still makes me nervous, but probably not worth the hassle to do things more safely.

Reply:
We need to do some debugging and figure out what the real issue is, but now didn't seem like the time.